src/main/resources/application.yml

```diff
@@ -17,6 +17,8 @@ spring:
     hikari:
       # time in ms to wait for db connection
       initialization-fail-timeout: 1800000
+      data-source-properties:
+        reWriteBatchedInserts: true
```

Comment:
This one should make a difference, even without hibernate. It makes it so that DB doesn't internally break batched inserts back up into single inserts when running.